contracts/upgradeable_contracts/erc20_to_erc20/HomeBridgeErcToErc.sol

```diff
@@ -61,11 +165,39 @@ contract HomeBridgeErcToErc is ERC677Receiver, EternalStorage, BasicBridge, Basi
 
     function onExecuteAffirmation(address _recipient, uint256 _value, bytes32 txHash) internal returns(bool) {
         setTotalExecutedPerDay(getCurrentDay(), totalExecutedPerDay(getCurrentDay()).add(_value));
-        return erc677token().mint(_recipient, _value);
+        uint256 valueToMint = _value;
+        address feeManager = feeManagerContract();
+        if (feeManager != address(0)) {
+            uint256 fee = calculateFee(valueToMint, false, feeManager, FOREIGN_FEE);
+            distributeFeeFromAffirmation(fee, feeManager);
+            emit FeeDistributedFromAffirmation(fee, txHash);
```

Comment:
We don't need these event anymore after https://github.com/poanetwork/poa-bridge-contracts/pull/176

Reply:
Thanks! Updated logic here https://github.com/poanetwork/poa-bridge-contracts/pull/166/commits/6b25ffb90c229567046c99b7f39caf3990bb8721